clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -242,6 +247,105 @@ private ResourcePageSource createPageSource(String poolName, ServerSideConfigura
     return pageSource;
   }
 
+  private void registerStatistics(String cacheAlias, PageSource pageSource) {
+
+    ServerStoreImpl store = stores.get(cacheAlias);
+
+    registerStatistic(pageSource, cacheAlias, "allocatedMemory" ,new Callable<Number>() {
```

Comment:
store stats should be registered  with a ServerStoreImpl context and the storeName